permissions/plugin-gitee.yml

```
@@ -8,5 +8,4 @@ paths:
 cd:
   enabled: true
 developers:
-  - "yashin"
-  - "markewaite"
+  - "puffy1215"
```

I'd prefer that we not drop the other maintainers without their agreement to be dropped.
```suggestion
  - "yashin"
  - "markewaite"
    - "puffy1215"
```

---

Of course